lightning/src/ln/channelmanager.rs

```diff
@@ -8741,7 +8765,7 @@ where
 	/// Creates a blinded path by delegating to [`MessageRouter::create_blinded_paths`].
 	///
 	/// Errors if the `MessageRouter` errors or returns an empty `Vec`.
```

Comment:
The empty `Vec` part of the docs is no longer accurate. Same below.

Reply:
Right!
Updated the docs in [4461115](https://github.com/lightningdevkit/rust-lightning/commit/4461115e88f7fb3e0a366a24aba8a32bc6c7b408)
Thanks for the pointer!